lib/api.common.js

```diff
@@ -657,39 +643,76 @@ api.common.eventEmitter = () => {
 
 api.common.cache = (
   // Extend Map interface total allocated size: map.allocated
+  options
 ) => {
   const cache = new Map();
+  const calcSize = (options && options.calcSize) || false;
```

Comment:
Never mind, I'd thought I was a number before I read the code further. Maybe it's indicating that the variable name isn't that good though.